addons/l10n_it_edi/models/account_move.py

```diff
@@ -0,0 +1,458 @@
+# -*- coding:utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import base64
+import logging
+import re
+
+from odoo import api, fields, models, _
+from odoo.tools import float_compare, float_repr
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    l10n_it_edi_state = fields.Selection([
+        ('being_sent', 'Being Sent To SdI'),
+        ('needs_signing', 'Needs signing'),
```

Comment:
In Italian "by the user" is hard to translate, it would be "Richiede firma da parte dell'utente"
"Requires user signature" maybe, it would be "Richiede firma (dell')utente". 
Whatcha think?